Caddyfile

```diff
@@ -0,0 +1,12 @@
+localhost:80 {
+	root * /usr/share/caddy
+	try_files {path} /index.html
+	file_server
+	#header @static Cache-Control "public, max-age=259200"
+	#encode zstd gzip
```

Comment:
I see no reason not to enable gzip compression.

Reply:
Is it necessary in a local-first, dev env?